src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/PetInfoOverlay.java

```diff
@@ -368,7 +369,8 @@ public static float getXpGain(Pet pet, float xp, String xpType) {
 	@Override
 	public void updateFrequent() {
 		Pet currentPet = getCurrentPet();
-		if (!NotEnoughUpdates.INSTANCE.config.petOverlay.enablePetInfo || currentPet == null) {
+		if (!NotEnoughUpdates.INSTANCE.config.petOverlay.enablePetInfo || currentPet == null ||
+			SBInfo.getInstance().getLocation() != null && SBInfo.getInstance().getLocation().equals("rift")) {
```

Comment:
Please place clarifying parenthesis around this. Alternatively: 

```suggestion
			Objects.equals(SBInfo.getInstance().getLocation(), "rift")) {
```